bw2io/importers/excel_lcia.py

```diff
@@ -30,12 +38,24 @@ class ExcelLCIAImporter(LCIAImporter):
 
     See the `documentation <https://2.docs.brightway.dev/intro.html#importing-lcia-methods-from-the-standard-excel-template>`__.
```

Comment:
@michaelweinold: follow-up!

Reply:
It was in the original code https://github.com/brightway-lca/brightway2-io/blob/documentation_improvements/bw2io/importers/excel_lcia.py#L31. It goes to legacy documentation https://documentation.brightway.dev/en/legacy/#importing-lcia-methods-from-the-standard-excel-template